src/content/blog/whats-new-february-2024.mdx

```diff
@@ -0,0 +1,212 @@
+- 🎙️ [Podrocket with Eddy Vinck](https://twitter.com/podrocketpod/status/1760351149440335877)by @eddyvinck
+- 📄 [Introduction to Astro and Wordpress](https://dev.to/bngmnn/leveraging-wordpress-as-a-headless-cms-for-your-astro-website-a-comprehensive-guide-a4d) by @bngmnn
+- 📄 [Building a documentation site using Astro Starlight](https://maciekpalmowski.dev/blog/building-a-documentation-site-using-astro-starlight/) by @palmiak
+- 📄 [A talk on Islands Architecture](https://js-astro-islands-demo.netlify.app/) by @winterjoey
+- 📄 [How to create animated images with TailwindCSS and Astro](https://lexingtonthemes.com/tutorials/how-to-create-animated-images-with-tailwind-css-and-astro-js/) by @mike_andreuzza
+- 📄 [Hands-On Experience: eCommerce Store with Astro?](https://crystallize.com/blog/building-ecommerce-with-astro)
+
+### Tips & Tools
+
+The latest community-built utilities and integrations to help you build with Astro.
+
+- [Unpic](https://unpic.pics/blog/unpic-astro-image-service/) by @ascorbic.
+- [florian-lefebvre/satori-astro](https://github.com/florian-lefebvre/satori-astro)
+- [astro-clerk-auth](https://www.npmjs.com/package/astro-clerk-auth) by @elefcodes
+- [Adammatthiesen/astro-commercejs](https://github.com/Adammatthiesen/astro-commercejs) by @adamm2047
+- [inline-mod-astro](https://inox-tools.vercel.app/inline-mod) by @fryuni
+- [JulianCataldo/og-images-generator](https://github.com/JulianCataldo/og-images-generator#astro-integration) by @juliancataldo
+- [BryceRussell/astro-pages](https://github.com/BryceRussell/astro-pages) by @bryceguy
+- [ktym4a/astro-page-insight](https://github.com/ktym4a/astro-page-insight) by @ktym4a
+- [kindspells/astro-sri-scp](https://www.npmjs.com/package/@kindspells/astro-sri-csp) by @andreu
+- [Tiny toolbar app to show Tailwind Breakpoints](https://elk.zone/m.webtoo.ls/@zero_gravitas@indieweb.social/111925333428424557)	
+- [frankstallone/astro-cube-boilerplate](https://github.com/frankstallone/astro-cube-boilerplate) by @frank_stallone
+- [KindSpells/astro-shield](https://github.com/KindSpells/astro-shield) by @castarco
+- [natemoo-re/astro-remote](https://github.com/natemoo-re/astro-remote)	by @nmoodev
+- [Generate e-mails with Astro](https://github.com/luckydye/astro-email/tree/main) by @luckydye
+
+### Themes & Templates
+
+Several new Astro themes were added & released this month. Try them out!
+
+- [Illustration based template](https://github.com/manulthanura/Illustration-Based-Portfolio) by @manulthanura
+- [Astroplate](https://astro.build/themes/details/astroplate/) by @tffarhad
+- [Quantum](https://cosmicthemes.com/themes/quantum/) by @lastchaos.
+- [event-edge.gerome.dev](https://event-edge.gerome.dev/) by @gerome.dev
+- [ScrewFast](https://github.com/mearashadowfax/ScrewFast) by @soslan
+- [MinimalStudio, theme built with Astro, TailwindCSS and Alpine](https://minimalstudio.vercel.app/) by @mike_andreuzza
+- [Resume website with TinaCMS](https://astro-resume02.vercel.app/) by @alpeshnakrani
```

Comment:
```suggestion
- { url: 'https://github.com/manulthanura/Illustration-Based-Portfolio', title: '@manulthanura'},
- { url: 'https://astro.build/themes/details/astroplate/', title: '@tffarhad'},
- { url: 'https://cosmicthemes.com/themes/quantum/', title: '@lastchaos.'},
- { url: 'https://event-edge.gerome.dev/', title: '@gerome.dev'},
- { url: 'https://screwfast.uk/', title: '@soslan'},
- { url: 'https://minimalstudio.vercel.app/', title: '@mike_andreuzza'},
- { url: 'https://astro-resume02.vercel.app/', title: '@alpeshnakrani'},
```

Reply:
astroplate, quantum, screwfast only have their preview/OG images linked 🤔 maybe this suggestion not good 😞

Reply:
Sadly yes, this will have to be another PR explicitly targeting the component to address this -- make sure the link goes to the repo but the preview comes from the demo, for example.